### ISSUE TITLE
Is it me or does Wiremod not work for me?

### ISSUE THREAD

**Issue by Zebrahead432** [Facepunch/garrysmod-issues#222]
![2013-07-03_00001](https://f.cloud.github.com/assets/4934225/746689/26d40232-e435-11e2-92d0-6e084c998764.jpg)
![gmod](https://f.cloud.github.com/assets/4934225/746694/6889a506-e435-11e2-8930-167efb0b477a.jpg)

So i use TortiseSVN and go to checkout and it installs everything then i put it in the addons folder then when i go into Singleplayer the WIRE bar doesnt show and everything else related to wire isnt there

In short i can't get wiremod to show or work
